decentraland/unity-explorer
# feat: Added nametag and chat bubble to own user

## What does this PR change?

Adds nametag and chat bubble to own user
...

## How to test the changes?



1. Launch the explorer
2. Chat a bit
3. Check the own nametag and chat bubbles

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
